atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandler.java

```diff
@@ -166,4 +220,16 @@ static boolean isFastFailoverException(Throwable ex) {
             + " Error writing to Cassandra socket."
             + " Likely cause: Exceeded maximum thrift frame size;"
             + " unlikely cause: network issues.";
+
+    @VisibleForTesting
+    enum Backoff {
```

Comment:
Right now this could just be a `boolean`, what's the use case you foresee where we would want different Backoff types?

Reply:
I also had a `LONG` behavior which was used for QoS exceptions, and would backoff for 10s. This is not needed anymore, since we're not going to retry QoS exceptions at this level, so converted back to a boolean.